# Cloud Label
Header header

sensor_msgs/PointCloud2 main_point_cloud
label_point[] main_label_cloud

sensor_msgs/PointCloud2 left_point_cloud
label_point[] left_label_cloud

sensor_msgs/PointCloud2 right_point_cloud
label_point[] right_label_cloud

sensor_msgs/PointCloud2 rear_point_cloud
label_point[] rear_label_cloud
